front/app/containers/Admin/projects/project/inputImporter/ReviewSection/IdeaEditor/utils.ts

```diff
@@ -76,16 +69,10 @@ export const isUserFormDataValid = (userFormData: UserFormData | null) => {
   if (!userFormData) return false;
   if (!userFormData.consent) return true;
 
-  if (['no-user', 'invalid-email'].includes(userFormData.user_state)) {
-    return false;
-  }
-
-  const { email } = userFormData;
-  const validEmail = email ? isValidEmail(email) : false;
-
-  if (!validEmail) return false;
+  const { email, first_name, last_name } = userFormData;
 
-  return true;
+  return ((email !== null && email !== undefined) ||
+    (first_name && last_name)) as boolean;
```

Comment:
```suggestion
  return !isNil(email) || !!(first_name && last_name);
```
or maybe even
```suggestion
  return !!email || !!(first_name && last_name);
```

Is the form valid if `email` is an empty string? Also, could it be safer to convert to boolean `!!` instead of `as`?

Reply:
No the form isn't valid with an empty string